### DISCUSSION TITLE
Cannot resolve file 'shim' and 'peer'

### DISCUSSION THREAD

**Discussion by DI-O** [hyperledger/fabric#2436] [Q&A] [answered]
wrong import, look this: https://github.com/hyperledger/fabric/blob/master/integration/lifecycle/testdata/chaincode/simple-v14/chaincode.go#L14

on master branch.

**Comment by lindluni** [comment, marked as answer]
This is a legacy chaincode meant only for doing integration testing on the `hyperledger/fabric-ccenv:1.4` image for testing migrations. This image contained the shim.

Future bugs like this should be directed to [Jira](https://jira.hyperledger.org)
